docs/builtin_elements.md

```diff
@@ -463,7 +463,8 @@ The FocusScope exposes callback to intercept the pressed key when it has focus.
 
 ```60
 Example := Window {
-    FocusScope {
+    forward-focus: MyKeyHandler;
+    MyKeyHandler := FocusScope {
```

Comment:
Good point. Maybe we should make that automatic: If one of element's children is focusable (because it's a FocusScope or uses forward-focus itself), then this should perhaps imply forwarding of focus to that child.

Reply:
Maybe indeed.

Or at least the error message when trying to setting the focus to it could hint at the forward-focus property

Reply:
Could you make it recursive?

Also if an element has more than one focusable child, would it take the first one?
Else adding a new focusable element would have the unexpected side effect of removing the automatic focus if the previously sole focusable element. If this happens deep in the tree, this could be an hard-to-catch regression.

Reply:
But then what would happen if I call `focus()` on the second element or one of its descendants while the first sibling has an implicit or explicit forward-focus?